integration/e2ecortex/client.go

```diff
@@ -358,3 +359,17 @@ func (c *Client) DeleteAlertmanagerConfig(ctx context.Context) error {
 
 	return nil
 }
+
+func (c *Client) PostRequest(url string, body io.Reader) (*http.Response, error) {
```

Comment:
I have not added a dedicated method for calling `delete_series` API here because it is not common usage. Adding a dedicated method would require changing the factory method for `Client` to accept the purger URL and in turn change all the usages to pass an empty string.